docs/core/install/dependencies.md

```diff
@@ -97,6 +113,35 @@ The requirements above are also required if you come across one of the following
 
 ::: zone pivot="os-linux"
 
+# [.NET Core 3.1](#tab/netcore31)
+
+.NET Core 3.1 treats Linux as a single operating system. There's a single Linux build (per chip architecture) for supported Linux distributions.
+
+.NET Core 3.1 is supported on the following Linux distributions/versions:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                             | Version               | Architectures    |
+| ------------------------------ | --------------------- | ---------------- |
+| Red Hat Enterprise Linux       | 6, 7, 8               | x64 |
+| CentOS                         | 7+                    | x64 |
+| Oracle Linux                   | 7+                    | x64 |
+| Fedora                         | 29+                   | x64 |
+| Debian                         | 9+                    | x64, ARM32, ARM64 |
+| Ubuntu                         | 16.04+                | x64, ARM32, ARM64 |
+| Linux Mint                     | 18+                   | x64 |
+| openSUSE                       | 15+                   | x64 |
+| SUSE Enterprise Linux (SLES)   | 12 SP2+               | x64 |
+| Alpine Linux                   | 3.10+                 | x64, ARM64 |
+
+For more information about .NET Core 3.1 supported operating systems, distributions, and lifecycle policy, see [.NET Core 3.1 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1-supported-os.md).
+
+For more information about how to install .NET Core 3.1 on ARM64 (kernel 4.14+), see [Installing .NET Core 3.0 on Linux ARM64](https://gist.github.com/richlander/467813274cea8abc624553ee72b28213).
```

Comment:
does this gist need to be updated to cover 3.1 as well?

Reply:
I've talked to Rich and we're going to port it to an article. But yes it's applicable to 3.0+